storage/redis/redis_scheduler_event_storage.go

```diff
@@ -0,0 +1,79 @@
+// maestro
+// https://github.com/topfreegames/maestro
+//
+// Licensed under the MIT license:
+// http://www.opensource.org/licenses/mit-license
+// Copyright © 2017 Top Free Games <backend@tfgco.com>
+
+package redis
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/go-redis/redis"
+	"github.com/topfreegames/extensions/redis/interfaces"
+	"github.com/topfreegames/maestro/models"
+)
+
+type RedisSchedulerEventStorage struct {
+	redisClient interfaces.RedisClient
+}
+
+func NewRedisSchedulerEventStorage(redisClient interfaces.RedisClient) *RedisSchedulerEventStorage {
+	return &RedisSchedulerEventStorage {
+		redisClient: redisClient,
+	}
+}
+
+func (this *RedisSchedulerEventStorage) PersistSchedulerEvent(event *models.SchedulerEvent) error {
```

Comment:
WDYT about keeping the events healthy by removing old entries (for example, doing a `ZREMRANGEBYRANK key MAXAGE -2`) while persisting in new events.

Reply:
I made something different here:

ZREMRANGEBYSCORE key -inf (<score of 14 days ago>